multiqc/modules/bases2fastq/plot_samples.py

```diff
@@ -0,0 +1,312 @@
+    <script> R1cyclenum = {r}</script>
+    """.format(
+        # Generate unique plot ID, needed in mqc_export_selectplots
+        id=report.save_htmlid("per_base_composition_plot"),
+        d=json.dumps(["bases2fastq", data]),
+        c=json.dumps(["bases2fastq", colorDict]),
+        r=r1r2_split,
+    )
+    plotName = "Per Cycle Sequence Composition"
+    anchor = "per_cycle_sequence_content"
+    description = "The proportion of each base position for which each of the four normal DNA bases has been called."
+    helptext = """
+    The heatmap is formatted in a similar way as fastqc per base sequence content. The differences are:
+    1) The pass/fail flag in the original fastqc module are replaced with a color reflecting the group tag of the samples
+    2) Read1 and Read2 is co-plotted and separate by a white band in the heatmap and a red dashed in each individual sample plot.
+
+    The following is a copy of helptext of the fastqc module:
+    To enable multiple samples to be shown in a single plot, the base composition data
+    is shown as a heatmap. The colours represent the balance between the four bases:
+    an even distribution should give an even muddy brown colour. Hover over the plot
+    to see the percentage of the four bases under the cursor.
+
+    **To see the data as a line plot, as in the original FastQC graph, click on a sample track.**
+
+    From the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/4%20Per%20Base%20Sequence%20Content.html):
+
+    _Per Base Sequence Content plots out the proportion of each base position in a
+    file for which each of the four normal DNA bases has been called._
+
+    _In a random library you would expect that there would be little to no difference
+    between the different bases of a sequence run, so the lines in this plot should
+    run parallel with each other. The relative amount of each base should reflect
+    the overall amount of these bases in your genome, but in any case they should
+    not be hugely imbalanced from each other._
+
+    _It's worth noting that some types of library will always produce biased sequence
+    composition, normally at the start of the read. Libraries produced by priming
+    using random hexamers (including nearly all RNA-Seq libraries) and those which
+    were fragmented using transposases inherit an intrinsic bias in the positions
+    at which reads start. This bias does not concern an absolute sequence, but instead
+    provides enrichement of a number of different K-mers at the 5' end of the reads.
+    Whilst this is a true technical bias, it isn't something which can be corrected
+    by trimming and in most cases doesn't seem to adversely affect the downstream
+    analysis._
+    """
+    plotContent = data
+    return html, plotName, anchor, description, helptext, plotContent
+
+
+def plot_per_cycle_N_content(sampleData, groupLookupDict, colorDict):
```

Comment:
Can the [FastQC code for that plot](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/fastqc/fastqc.py#L693) be adapted here?